news/_posts/2020-03-20-ps17-patch-release-lifecycle.md

```diff
@@ -0,0 +1,126 @@
+
+Then maintainers start working on fixing them (or merging the bug fixes submitted by the community).
+
+Obviously, security issues are not processed the same way: when a vulnerability is reported, it is being explored and it is being fixed in a hidden manner in order to make sure hackers unaware of the vulnerability do not hear about it. We use [GitHub Security Advisories](https://help.github.com/en/github/managing-security-vulnerabilities/about-github-security-advisories) and temporary private forks to collaborate on the fix. Maintainers only publish the advisory and the fix on the day of the release[(2)](#glossary), following [responsible disclosure](https://en.wikipedia.org/wiki/Responsible_disclosure) principle.
+
+When all of the bug fixes for the target patch version are merged, and all teams pressure themselves to make it happen before the end of the 6 weeks, maintainers deliver a Release Candidate to QA team for the standard patch release test campaign. This campaign aims to find whether this patch introduces new bugs.
+
+If the campaign reports that no bugs are found, the new patch release is delivered!
+
+## Why maintain only the latest minor version ?
+
+As explained above, PrestaShop only maintains one minor version.
+When PrestaShop 1.7.7.0 is out, PrestaShop 1.7.6 will be considered [End Of Life](https://en.wikipedia.org/wiki/End-of-life_(product)) and not maintained anymore. No more bug fixes for this version, no more patch releases, not even security releases[(2)](#glossary).
+
+This project management rule might be seen as harsh, because it means that if you do not upgrade to PrestaShop 1.7.7.0, you shop will use an outdated and unmaintained version and become possibly vulnerable to newly found security issues.
+
+However this rule is actually dictated by the current very wide scope of the PrestaShop project. Very often, maintainers are working both on a patch release and the next minor release. For example one week ago work was happening on both 1.7.6.5 patch release development and 1.7.7.0 Feature Freeze. The scope of PrestaShop project is very wide which is why maintainers need to focus their efforts on the most important components, so the scope ony contains these two versions.
+
+We are working to make the maintainers team grow (and be able to handle a wider scope) with not only new PrestaShop employees, [but also with external people](https://github.com/PrestaShop/open-source/issues/11).
+When the team grows, it could be able to provide patch releases on more versions, like two or three of them, or provide [Long Term Support](https://en.wikipedia.org/wiki/Long-term_support) releases.
+
+### Six weeks is a long time: if a major bug is reported, merchants need to wait up to six weeks before being provided a patched version
```

Comment:
I am afraid we are giving the stick to be beaten with here, wdyt ?

Reply:
I agree. Plus, 6 weeks is not long at all in the software industry for a patch release.

Reply:
We can improve the sentence 😄

I wanted to explain the reasoning between the 6 weeks. There might be a better way to introduce it.

Reply:
how does it compare to Google's recommendations? 90 days or something like that?

Reply:
@ttoine I dont know if there are such recommendations.

I chose a sentence a lot more simpler for this part "Why six weeks, and not eight or two ?" 
https://github.com/PrestaShop/prestashop.github.io/pull/667/commits/d589875ec49beb6fd709e621c8f5667e151b9d6b